llvm/lib/Passes/PassBuilderPipelines.cpp

```diff
@@ -1825,6 +1827,7 @@ PassBuilder::buildLTODefaultPipeline(OptimizationLevel Level,
     // in ICP.
     MPM.addPass(LowerTypeTestsPass(nullptr, nullptr,
                                    lowertypetests::DropTestKind::Assume));
+    MPM.addPass(CoroCleanupPass());
```

Comment:
am I reading the code correct as in: For a `-O0` build, we would only have the `CoroEarly` and the `CoroCleanup` pass? We only add the `CoroSplitPass` for `-O2`? Afaik, we do need the `CoroSplitPass` also for `O0` and `O1`

Reply:
I assumed they are option optimizations, will add them